clitests/tests/encode/encode_astc.json

```diff
@@ -0,0 +1,54 @@
+{
+    "description": "Test default behavior of UASTC encoding.",
```

Comment:
The description should say ASTC.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software-CTS/pull/33/commits/bb7cdf30e166ecfed1ec605289073f7a7eec471b